src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -183,8 +203,8 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
   };
 
   const articleBlocks = startsWithHeading
-    ? blocks
-    : [visuallyHiddenBlock, ...blocks];
+    ? transformedBlocks
```

Comment:
I'm a bit confused on what's happening here. Does this always add in the experiment?

Reply:
I think it might be. This was flagged by a TS error I get on line 102

`Right operand of ?? is unreachable because the left operand is never nullish.`

If I add something in like the below on line 205 then the unit tests pass

```
  const blocksToUse = shouldEnableExperimentTopStories
    ? transformedBlocks
    : blocks;
```

Reply:
I think the function now returns either the blocks unchanged _or_ the experimentally changes to the blocks. This should probably be made more clear by updating the name of the function to de-couple it from experimentation. i.e `getExperimentTopStoriesBlocks` should really be `getTopStoriesBlocks`

Reply:
@iainrj I've kept the name `getExperimentTopStoriesBlocks` since it doesn't get the data for the original `TopStoriesSection` in the secondary column and because the top stories blocks in the article blocks array have `type: experimentTopStories`.

Reply:
Nice one, thanks :)